### ISSUE TITLE
delete one card should not skip related cards

### ISSUE THREAD

**Issue by HQYang1979** [mortii/anki-morphs#77]
![image](https://github.com/mortii/anki-morphs/assets/74698160/917d0115-d550-4b8d-aa10-7ada35c49ea9)

delete one card should not skip related cards.

I see that when I delete one card (because of bad audio or bad sentence), it shows skipping multiple cards. I am not sure what cards are skipped, but I assume they are related cards. I think only K can skip related cards.

**Comment by Vilhelm-Ian** [comment]
@HQYang1979 it's not skipping related cards. It's just doing something like the recalc button does

**Comment by HQYang1979** [comment]
can it be changed to recalc ing, in order not to cause confusion?

**Comment by mortii** [comment]
> I am not sure what cards are skipped, but I assume they are related cards. 

No, AnkiMorphs does not update anything (seen morphs, etc.) when a card is deleted.

There are three ways cards can get skipped after a card is deleted:
1. The unknown morph on the next card was a name you added to names.txt
2. The next card has one of the 'known' tags (`am-known-manually` or `am-known-automatically`) 
3. You have already seen the unknown morph on the next card, but it was sorted to be after the card you deleted. This happens often; if you use 'L' -key on a card you can see that cards with the same unknown morphs are often not right after each other (the 'New #.....' numbers) .

**Comment by HQYang1979** [comment]
Good to know, thank you very much!
